app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+require('dotenv').config({ path: './config/test.env' }); // Default: path.resolve(process.cwd(), '.env')
```

Comment:
Please dont use this line of code twice. Use it in the begining of the index.js just once.,